specs/commandline/sign.md

```diff
@@ -37,6 +37,48 @@ Flags:
       --plugin-config strings    {key}={value} pairs that are passed as it is to a plugin, refer plugin's documentation to set appropriate values
       --signature-format string  signature envelope format, options: 'jws', 'cose' (default "jws")
   -u, --username string          username for registry operations (default to $NOTATION_USERNAME if not specified)
+      --use-image-manifest       optional for using OCI image manifest to store signatures in a registry
+```
+
+## Use OCI image manifest to store signatures
+
+By default, Notation uses [OCI artifact manifest][oci-image-manifest] to store signatures in the registries. For backward compatibility, Notation supports using `OCI image manifest` to store the signature in the registries that implement partial of the OCI Image specification v1.1. Use flag `--use-image-manifest` to force Notation store the signatures using OCI image manifest.
+
+Registries MAY not implement or enable the `Referrers API`, which is used by the client to fetch the referrers. In the context of Notation, the referrers are signatures. Notation follows the fallback procedure defined in [OCI distribution spec][oci-backward-compatibility] if `Referrers API` is unavailable.
+
+### Set config property for OCI image manifest
+
+OCI image manifest requires additional property `config` of type `descriptor`, which is not required by OCI artifact manifest. Notation creates a default config descriptor for the user if the flag `--use-image-manifest` is used.
+
+Notation uses empty JSON object `{}` as the default configuration content, and thus the default `config` property is fixed, as following:
+
+```json
+"config": {
+    "mediaType": "application/vnd.cncf.notary.signature",
+    "size": 2,
+    "digest": "sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a"
+}
+```
+
+### When to use OCI image manifest
+
+[Registry support][registry-support] lists registries with different compatibilities. For registries not supporting `OCI artifact manifest`, user can use flag `--user-image-manifest` to sign artifacts stored in those registries.
+
+In case registries not listed in the page, user can consider using flag `--user-image-manifest` by checking the error message. Note that there is no deterministic way to determine a registry supporting `OCI artifact manifest` or not. The error message is just for reference. The following response status contained in error messages MAY indicate that the registries doesn't support `OCI artifact manifest`:
+
+- Response status `400 BAD Request` with error code `MANIFEST_INVALID` or `UNSUPPORTED`
+
+### Copy signatures between registries
+
+User may use tools to copy artifacts and associated signatures between two registries. The copy succeeds under the following conditions:
+
+- From registries that support `OCI artifact manifest` to registries that support `OCI artifact manifest`
+- From registries that support `OCI image manifest` to registries that support `OCI image manifest`
```

Comment:
We might want to move the `Copy signatures between registries` section out of this spec, because it is named sign.md and it's for `notation sign` CLI command. Perhaps notaryproject repo is a better place for this section?